web-tier/src/main/java/com/outdoor/foodcalc/model/dish/DishProduct.java

```diff
@@ -1,25 +1,18 @@
 package com.outdoor.foodcalc.model.dish;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.validation.constraints.Min;
 import javax.validation.constraints.PositiveOrZero;
 import java.util.Objects;
 
-public class DishProduct {
+@AllArgsConstructor
```

Comment:
https://projectlombok.org/features/Data

AllArgsConstructor in included in Data, please try to remove it
and double check if NoArgsConstructor requires explicit declaration

and add Builder

Reply:
they both require an explicit declaration, there are errors without any of them

Reply:
Builder added